canvas_modules/harness/test_resources/parameterDefs/toggle_paramDef.json

```diff
@@ -0,0 +1,507 @@
+     "string_error": [
+      "testing",
+      "toggle",
+      "control"
+    ],
+    "string_warning": [
+      "testing",
+      "toggle",
+      "control"
+    ],
+    "hide": true,
+    "string_hidden": [
+      "testing",
+      "toggle",
+      "control"
+    ],
+    "disable": true,
+    "string_disabled": [
+      "testing",
+      "toggle",
+      "control"
+    ],
+    "string_table": [
+      [
+        [
+          "Hopper"
+        ],
+        [
+          "Turing"
+        ],
+        [
+          "Shannon"
+        ],
+        [
+          "Babbage"
+        ]
+      ],
+      []
+    ],
+    "string_table_cond": [
+      [
+        [
+          "Hopper"
+        ],
+        [
+          "Turing"
+        ],
+        [
+          "Shannon"
+        ],
+        [
+          "Babbage"
+        ]
+      ],
+      []
+    ]
+  },
+  "parameters": [
+    {
+      "id": "string",
+      "type": "string"
+    },
+    {
+      "id": "string_default",
+      "type": "array[string]",
+      "default": "default testing"
+    },
+    {
+      "id": "string_empty",
+      "type": "array[string]",
+      "default": "ERROR - should not be set"
+    },
+    {
+      "id": "string_undefined",
+      "type": "array[string]"
+    },
+    {
+      "id": "string_null",
+      "type": "array[string]",
+      "default": "ERROR - should not be set"
+    },
+    {
+      "id": "string_error",
+      "type": "array[string]"
+    },
+    {
+      "id": "string_warning",
+      "type": "array[string]"
+    },
+    {
+      "id": "hide",
+      "type": "boolean"
+    },
+    {
+      "id": "string_hidden",
+      "type": "array[string]",
+      "required": true
+    },
+    {
+      "id": "disable",
+      "type": "boolean"
+    },
+    {
+      "id": "string_disabled",
+      "type": "array[string]",
+      "required": true
+    },
+    {
+      "id": "string_table",
+      "type": "array[toggles]"
+    },
+    {
+      "id": "string_table_cond",
+      "type": "array[toggles]"
+    },
+    {
+      "id": "string_long_value",
+      "type": "array[string]"
+    }
+  ],
+  "complex_types": [
+    {
+      "id": "toggles",
+      "parameters": [
+        {
+          "id": "toggle",
+          "type": "array[string]"
+        },
+        {
+          "id": "toggle2",
+          "type": "array[string]"
+        },
+        {
+          "id": "toggle_on_panel",
+          "type": "array[string]"
+        },
+        {
+          "id": "toggle_sub_panel",
+          "type": "array[string]"
+        }
+      ]
+    }
+  ],
+  "uihints": {
+    "id": "toggle",
+    "icon": "images/default.svg",
+    "label": {
+      "default": "string Fields"
+    },
+    "parameter_info": [
+      {
+        "parameter_ref": "string",
+        "label": {
+          "default": "string"
+        },
+        "control": "toggle",
+        "description": {
+          "default": "toggle with sting value"
+        },
+        "class_name": "string-toggle-control-class"
```

Comment:
Please add `toggleonlabel` and `toggleofflabel` in paramDef for testing.

Reply:
<img width="548" alt="image" src="https://user-images.githubusercontent.com/89609176/182446041-da3adb19-ec47-4e3f-a26a-1e125fd843e9.png">

Reply:
Was this pushed. This wouldn't be the correct way of handling labels since they need to be translatable.